src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -59,9 +59,7 @@ namespace CoordinateMaps {
  *
  *  with derivatives:
  *
- *  \f[\Xi'(\xi) = 1\f]
- *
- *  \f[\mathrm{H}'(\eta) = 1\f]
+ *  <center>\f$\Xi'(\xi) = 1\f$, and \f$\mathrm{H}'(\eta) = 1\f$</center>
```

Comment:
somewhere in the documentation move the two "Where"s out of the math and lowercase them